.github/workflows-src/steps.lib.yml

```diff
@@ -112,7 +113,10 @@ env:
 #@ def steps_inttest():
 name: "Integration Tests"
 shell: bash
-run: SHELL='' go test -timeout 20m ./test/integration -v
+run: |
+  export SHELL=''
+  mkdir -p logs
+  go test -timeout 20m ./test/integration -v
```

Comment:
Can we get rid of this stuff and instead just echo the log file after running the job. The job outputs ARE the logs. This is overcomplicating things.

Reply:
I am mostly sure that that would mean a failure in the tests will result in no msi log output. Is there a simpler mechanism to ensure those logs will print? It's not clear if TRAP would work, and I'm out of other ideas at the moment.

Reply:
Just capture the exit code, do the print and then exit with the exit code? Should be way cheaper than all the stuff going on in this PR to produce logs as artifacts.

Reply:
Heck just a `set +e` might actually suffice

Reply:
Yeah, I think that would work. I don't know why I thought that was already the case and some other mechanic was in play.